download_mac.sh

```diff
@@ -2,4 +2,10 @@
 
 sudo apt-get install -qq sshpass
 gem install package_cloud
+
+# Download the mac library.
 sshpass -p $SSH_PASS sftp -oStrictHostKeyChecking=no -oPort=2223 -r $SSH_USER@185.19.29.18:/upload down/
+# Cleanup.
+sshpass -p $SSH_PASS sftp -oStrictHostKeyChecking=no -oPort=2223 $SSH_USER@185.19.29.18:/upload <<< $'rm 'libtraildbjava-$TRAVIS_TAG'.dylib'
```

Comment:
The syntax with the simple quotes looks suspicious... Is this command working ?
But if you tested and it works, fine :)